pwndbg/color/disasm.py

```diff
@@ -1,27 +1,33 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import capstone
-
 import pwndbg.chain
-import pwndbg.color
 import pwndbg.disasm.jump
+import pwndbg.config as config
+import pwndbg.color.memory as M
+import pwndbg.color.theme as theme
+from pwndbg.color import generateColorFunction, green, red
+
+import capstone
 
 capstone_branch_groups = set((
-capstone.CS_GRP_CALL,
-capstone.CS_GRP_JUMP
+    capstone.CS_GRP_CALL,
+    capstone.CS_GRP_JUMP
 ))
 
+config_branch = theme.Parameter('disasm-branch-color', 'bold', 'color for disasm (branch/call instruction)')
+
+def branch(x):
```

Comment:
oh mis,interpated the diff. this was moved to own module ``pwndbg/color/disasm.py`` and i think that is a good place for it. later if the soon following PR will add syntax highlighting I'm planning to provide disasm color override values in this file (``pwndbg/color/disasm.py``) to have own special colors for things like call, jmp, ret